README.md

```diff
@@ -1,21 +1,23 @@
+# Ninja
+
 Ninja is a small build system with a focus on speed.
 https://ninja-build.org/
 
-See the manual -- https://ninja-build.org/manual.html or
-doc/manual.asciidoc included in the distribution -- for background
+See [the manual](https://ninja-build.org/manual.html) or
+`doc/manual.asciidoc` included in the distribution for background
 and more details.
 
 Binaries for Linux, Mac, and Windows are available at
-  https://github.com/ninja-build/ninja/releases
-Run './ninja -h' for Ninja help.
+  [GitHub](https://github.com/ninja-build/ninja/releases).
+Run `./ninja -h` for Ninja help.
 
 To build your own binary, on many platforms it should be sufficient to
-just run `./configure.py --bootstrap`; for more details see HACKING.md.
-(Also read that before making changes to Ninja, as it has advice.)
+just run `./configure.py --bootstrap`; for more details see
```

Comment:
You just merged in CMake support.

I have not looked at the newly merged code, so it's not clear to me if this advice is still accurate. Might be that it is and nothing needs to be changed.

However, I suggest specifically mentioning the ability to use CMake to compile the ninja codebase here, as well as configure.py (or instead of configure.py)

Reply:
> I have not looked at the newly merged code, so it's not clear to me if this advice is still accurate.

It is :)

> However, I suggest specifically mentioning the ability to use CMake to compile the ninja codebase here, as well as configure.py (or instead of configure.py)

Good idea. Let's do that in another PR though (or just in the wiki first).